corehq/apps/app_manager/templates/app_manager/partials/case_list_lookup.html

```diff
@@ -122,9 +123,7 @@
                         {% trans "Results display text" %}
                     </label>
                     <div class="col-sm-4">
-                        <input class="form-control" type="text"
-                               data-bind="value: lookup_field_header"
-                               placeholder="{% trans 'Accuracy' %}" />
+                        {% input_trans detail.short.lookup_field_header langs 'lookup_field_header' data_bind='value: lookup_field_header' %}
```

Comment:
@orangejenny I'm trying this out for size. I realised that using `input_trans` doesn't give me the `lang-text` info button for free -- unless I'm missing something. I couldn't see how to get it in a simple, dry way.